index.js

```diff
@@ -803,4 +803,20 @@ export const mentorshipTeam = {
       },
     ],
   },
+  linhtran: {
+    name: 'Trang Linh',
+    title: 'Student @ Hobart and William Smith Colleges',
+    hobbies: 'draw, sleep, axolotl',
+    avatar:
+      'https://drive.google.com/file/d/1PNcVjP0bCi0IORUDy-VXamPnWJ7BymCh/view?usp=sharing',
```

Comment:
let's change to this url `https://res.cloudinary.com/khoa165/image/upload/v1711839128/viettech/tranglinh.jpg`